experimental/c-cpp/manager.py

```diff
@@ -474,31 +460,30 @@ def get_fuzzer_intrinsics(self, func: Dict[str, Any]) -> Dict[str, Any]:
 
     type_constraints = 'the types of types function are:\n'
     for idx, arg in enumerate(func['debug_function_info']['args']):
-      type_constraints += '- Argument %d is of type \"%s\"\n' % (idx + 1, arg)
+      type_constraints += f'- Argument {idx+1} is of type \"{arg}\"\n'
     type_constraints += (
         'You must make sure the arguments passed to the function match the ' +
         'types of the function. Do this by casting appropriately.')
 
     logger.info('Sample targets:')
-    prompt = '''Hi, please write a fuzz harness for me.
+    prompt = f'''Hi, please write a fuzz harness for me.
 
-The target project is %s which is a open source project written in C. The harness you write should be in pure C as well.
+The target project is {self.github_url} which is a open source project written in C. The harness you write should be in pure C as well.
 
-I would like for you to write the harness targeting the function `%s.`
+I would like for you to write the harness targeting the function `{func['function_signature']}.`
 
 The harness should be in libFuzzer style, with the code wrapped in `int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)`.
 
 Please wrap all code in <code> tags and you should include nothing else but the code in your reply. Do not include any other text.
 
 Make sure the ensure strings passed to the target are null-terminated.
 
-There is one rule that your harness must satisfy: all of the header files in this library is %s. Make sure to not include any header files not in this list.
+There is one rule that your harness must satisfy: all of the header files in this library is {str(headers_to_include)}. Make sure to not include any header files not in this list.
```

Comment:
Same above
Do we need to cast headers_to_include explicitly with str()?

Reply:
BTW, could this be in a template file in the final version?
Also, could we define repeated lines once and reuse them to avoid repetitions?

Reply:
> Same above Do we need to cast headers_to_include explicitly with str()?

We don't! thanks, fixed!

Reply:
> BTW, could this be in a template file in the final version? Also, could we define repeated lines once and reuse them to avoid repetitions?

Yep, I'll look at doing this when merging the two https://github.com/google/oss-fuzz-gen/issues/450 -- for the moment it's nice to have this in a single module as it makes for easy copying in/out of the containers as some development of this module happens within base-builder container environments. I will address this later though.